MAUI/PDF-Viewer/Printing-a-Document.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Printing PDF files in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about printing feature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Printing PDF Files in .NET MAUI PDF Viewer (SfPdfViewer)
+
+The print feature of [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to effortlessly print PDF documents directly from your application code.
+
+Moreover, when performing a print operation, the default device print dialog opens, providing users with familiar options and settings to customize their printing experience.
+
+To print a PDF programmatically, you can use the `PrintDocument` method provided by [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html).Refer to the following code example.
+
+{% tabs %}
+{% highlight c# %}
+// Prints the PDF document.
+PdfViewer. PrintDocument();
+{% endhighlight %}
+{% endtabs %}
+
+Also, SfPdfViewer provides another way to print through command binding using the `PrintDocumentCommand`. This makes printing easier, especially when used with buttons.
```

Comment:
APIs need to be linked. 
<img width="345" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/9af8eda3-b191-4761-91b2-f46df1d7a9bb">

Reply:
Linked the API.